README.md

```diff
@@ -1,2 +1,263 @@
 # blackbird-utils
 Helpers for React projects
+
+## Installation
+```
+npm install --save @blackbird/utils
```

Comment:
I "optimistically" spec'd the package name here as `@blackbird/utils`. From my rough understanding, by using an "org" package name, it'll allow more than one of us to be able to actually publish new versions of the package to npm

So I think creating an org (`@blackbird`?) would be a first step in setting up that publishing workflow